src/jvm/code/method_body.rs

```diff
@@ -123,6 +123,12 @@ impl<I> InstructionList<I> {
         self.0.first_key_value()
     }
 
+    /// Returns the last instruction in the list.
+    #[must_use]
+    pub fn exit_point(&self) -> Option<(&ProgramCounter, &I)> {
```

Comment:
This function returns the node with largest PC. However it may not be the exit point of a method.
- A method may not have exactly one exit point. For example, a method containing an infinite loop does not have an exit point, and a method with multiple `return` statements has multiple exit points.
- The exit points of a method should be the nodes in the control flow graph that does not have outgoing edges. You may check [`ControlFlowGraph::exits()`](https://docs.rs/mokapot/latest/mokapot/ir/struct.ControlFlowGraph.html#method.exits) for more details.

Reply:
I agree, I think the name of the function should be changed then, as my intent with this was to mirror the `entry_point` function - by returning that last node despite what it contained.

It sounds like a different function name should be put in place then.